demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -13,6 +13,12 @@
 class PlaybookValidator(ContentEntityValidator):
     """PlaybookValidator is designed to validate the correctness of the file structure we enter to content repo."""
 
+    def __init__(self, structure_validator, ignored_errors=None, print_as_warnings=False, json_file_path=None,
+                 validate_all=True):
```

Comment:
Why is the default True?

Reply:
You are right this is better to be False forgot to change this after a change i done in the code.